src/emails/templates/org-invite.tsx

```
@@ -0,0 +1,154 @@
+    );
+  } else if (themeName === "ol-data-platform") {
+    return (
+      <EmailLayout preview={`Invitation to MIT Open Learning Data Platform`} locale={locale}>
+        <Text style={paragraph}>
+          <p>
+            Hi,
+          </p>
+
+          <p>
+            You have been invited to join <span style={emphasizedText}>MIT Open Learning Data Platform</span>. 
+            This platform provides tools for data analysis and insights into learning patterns across MIT's courses.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)